.github/workflows/scripts/mark-issues-as-stale.sh

```diff
@@ -74,7 +74,7 @@ for ISSUE in ${ISSUES}; do
         echo "Pinging code owners for issue #${ISSUE}."
 
         # The GitHub CLI only offers multiline strings through file input.
-        printf "${STALE_MESSAGE}\n\nPinging code owners:\n${OWNER_MENTIONS}\nSee [Adding Labels via Comments](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-labels-via-comments) if you do not have permissions to add labels yourself." \
+        printf "%s $STALE_MESSAGE\n\nPinging code owners:\n%s $OWNER_MENTIONS\nSee [Adding Labels via Comments](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-labels-via-comments) if you do not have permissions to add labels yourself." \
```

Comment:
I would have expected `$STALE_MESSAGE` and `$OWNER_MENTIONS` to be at the very end of the `printf` command, could you explain why you did it like this?

Reply:
You are right!